### ISSUE TITLE
Matrix on the mobile. No titles

### ISSUE THREAD

**Issue by mikeamelin** [surveyjs/survey-library#7225]
![Image](https://github.com/surveyjs/survey-library/assets/17025812/cf881c13-e781-47c5-b949-4d48d040629c)

**Comment by JaneSjs** [comment]
The issue is reproduced on a [Matrix Rubric](https://surveyjs.io/form-library/examples/questiontype-matrix-rubric/reactjs#)

**Comment by JaneSjs** [comment]
As a w/a, use Matrix Dropdown
```
  {
     "type": "matrixdropdown",
     "name": "Clothing Size Chart",
     "minWidth": "256px",
     "title": "SIZE CHART",
     "description": "All sizes are in inches.",
     "descriptionLocation": "underTitle",
     "defaultValue": {
      "Row 2": {
       "Column 2": "14.5",
       "Column 3": "34",
       "Column 4": "35-37",
       "Column 5": "29.5–31.5"
      },
      "Row 3": {
       "Column 2": "15–15.5",
       "Column 3": "34.5–35",
       "Column 4": "38–40",
       "Column 5": "32.5–34.5"
      },
      "Row 4": {
       "Column 2": "16–16.5",
       "Column 3": "35.5–36",
       "Column 4": "42–44",
       "Column 5": "36.5–38.5"
      },
      "Row 5": {
       "Column 2": "17–17.5",
       "Column 3": "36.5–37",
       "Column 4": "46–48",
       "Column 5": "40.5–42.5"
      },
      "Row 6": {
       "Column 2": "18–18.5",
       "Column 3": "37.5–38",
       "Column 4": "50–52",
       "Column 5": "44.5–46.5"
      }
     },
     "alternateRows": true,
     "columns": [
      {
       "name": "Column 2",
       "title": "Neck"
      },
      {
       "name": "Column 3",
       "title": "Sleeve"
      },
      {
       "name": "Column 4",
       "title": "Chest"
      },
      {
       "name": "Column 5",
       "title": "Waist"
      }
     ],
     "choices": [
      1,
      2,
      3,
      4,
      5
     ],
     "cellType": "expression",
     "rows": [
      {
       "value": "Row 2",
       "text": "S"
      },
      {
       "value": "Row 3",
       "text": "M"
      },
      {
       "value": "Row 4",
       "text": "L"
      },
      {
       "value": "Row 5",
       "text": "XL"
      },
      {
       "value": "Row 6",
       "text": "XXL"
      }
     ],
     "rowTitleWidth": "64px"
    }
   ]
  }
```

**Comment by andrewtelnov** [comment]
This issue has been fixed in the last version, 1.9.118.

Thank you,
Andrew
